modules/account-lib/src/mpc/tss.ts

```diff
@@ -36,7 +36,7 @@ import Shamir from './shamir';
 import { randomBytes as cryptoRandomBytes } from 'crypto';
 
 const convertObjectHexToBuffer = (object: Record<string, any>, keys: string[]) => {
-  const result: any = object;
+  const result: any = { ...object };
```

Comment:
Change needed to make a deep copy of the object since it was just doing a reference and overwriting the object itself.